x-pack/plugins/apm/server/lib/helpers/create_es_client/call_async_with_debug.ts

```diff
@@ -70,3 +58,17 @@ export async function callClientWithDebug({
 
   return res;
 }
+
+export const getSearchDebugBody = (params: Record<string, any>) =>
+  `GET ${params.index}/_search\n${formatObj(params.body)}`;
+
+export const getDefaultDebugBody = (
+  params: Record<string, any>,
+  operationName: string
+) =>
+  `${chalk.bold('ES operation:')} ${operationName}\n${chalk.bold(
+    'ES query:'
+  )}\n${formatObj(params)}`;
```

Comment:
nit: perhaps a co-locating the debug message in a single function will make it easier to understand when one is used over the other:

```suggestion
export const getDebugBody = (
  params: Record<string, any>,
  operationName: string
) => {
  if (operationName === 'search') {
    return `GET ${params.index}/_search\n${formatObj(params.body)}`;
  }

  // message for all but 'search' operatins
  return `${chalk.bold('ES operation:')} ${operationName}\n${chalk.bold(
    'ES query:'
  )}\n${formatObj(params)}`;
}
```